Web3Swift/Account/EthAutoAccount.swift

```diff
@@ -0,0 +1,76 @@
+//
+// Created by Timofey on 3/19/18.
+//
+
+import Foundation
+import SwiftyJSON
+
+//Eth account that asks gas price and gas estimate from the network
+public final class EthAutoAccount: Account {
+
+    private let network: Network
+    private let privateKey: PrivateKey
+
+    /**
+    Ctor
+
+    - parameters:
+        - network: network to work with
+        - privateKey: private key associated with the account
+    */
+    init(
+        network: Network,
+        privateKey: PrivateKey
+    ) {
+        self.network = network
+        self.privateKey = privateKey
+    }
+
+    /**
+    - returns:
+    Balance of the account in wei
+
+    - throws:
+    `DescribedError` if something went wrong
+    */
+    public func balance() throws -> NumberScalar {
+        return try EthBalance(
+            network: network,
+            address: privateKey.address()
+        )
```

Comment:
@biboran should it be `.value()` here?

Reply:
@rockfridrich no, it is intended to be lazy